AtDigitization/AtVectorResponse.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ATVECTORRESPONSE_H
+#define ATVECTORRESPONSE_H
+
+#include "AtDigiPar.h"
+
+#include <FairRunAna.h>
+#include <FairRuntimeDb.h>
+#include <FairTask.h>
+
+#include <Rtypes.h>
+#include <TString.h>
+
+#include <cmath>
+#include <cstdio>
+#include <iostream>
+#include <vector>
```

Comment:
I think this should be the only header file here (along with Rtypes.h). Everything else is not used in the class definition

Reply:
This is pointing to the vector include.